Candidature: Prévenir dans la candidature que la personne a quitté l’organisation (#4916)
gip-inclusion/les-emplois: itou/templates/layout/base_email_text_body.txt

{% autoescape off %}

{% if forward_from_user|default:False %}

> Vous recevez cet e-mail parce que l'utilisateur {{ forward_from_user.get_full_name }} ({{ forward_from_user.email}}) ne fait plus partie de votre organisation

Reviewer: ```suggestion
Vous recevez cet e-mail parce que l'utilisateur {{ forward_from_user.get_full_name }} ({{ forward_from_user.email}}) ne fait plus partie de votre organisation.
```

> Vous recevez cet e-mail parce que l'utilisateur {{ forward_from_user.get_full_name }} ({{ forward_from_user.email}}) ne fait plus partie de votre {% if forward_from_user.kind == "employer" %}structure{% else %}organisation{% endif %}.

Reviewer: ```suggestion
Vous recevez cet e-mail parce que l'utilisateur {{ forward_from_user.get_full_name }} ({{ forward_from_user.email}}) ne fait plus partie de votre {{ forward_from_user.is_employer|yesno:"structure,organisation" }}.
```
